.claude/skills/nerbis-design-identity/rules/onboarding.md

```
@@ -0,0 +1,193 @@
+```
+
+## Empty State → First Action Bridge
+
+Every section the user visits during onboarding should have a **contextual empty state** that bridges to the first action:
+
+```tsx
+// Products page — first visit
+<EmptyState
+  icon={<IconPackage className="size-10 stroke-1 text-muted" />}
+  title="No products yet"
+  description="Add your first product — it takes about 2 minutes."
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)